### ISSUE TITLE
You have an bug at Playlist.py

### ISSUE THREAD

**Issue by DixitSom** [spotify-api/spotify-api.py#8]
Hello should be closed in parenthesis but its open and thronging an EOF error.

**Comment by Scientific-Guy** [comment]
Yeah, spotify-api.py is broken. And we are working on making the new one with typings and structures soon.
I am not sure when it will release but it will release near end of the month or around it.
You can submit a pull request if you can help us...

**Comment by DixitSom** [comment]
Okay I will do it.

**Comment by scientific-dev** [comment]
And sorry for the delay. The work is restarted again and will be completed in some time i guess.
